internal/pkg/cli/cli.go

```diff
@@ -107,3 +107,19 @@ func relativeDockerfilePath(ws copilotDirGetter, dockerfilePath string) (string,
 	}
 	return relDfPath, nil
 }
+
+// dfBuildRequired returns if the container image should be built from local Dockerfile.
+func dfBuildRequired(svc interface{}) (bool, error) {
```

Comment:
I wonder if this should be a function in the `manifest` pkg: `manifest.ImgRequiresBuild() (bool, error)`

Reply:
sounds good!